cypress/integration/learn/index.js

```diff
@@ -48,3 +48,159 @@ describe('Learn Landing page', function() {
     });
   });
 });
+
+describe('Quotes', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.contains("Get started (it's free)").click({ force: true });
+  });
+
+  it('Should show a quote', () => {
+    cy.get('blockquote').within(() => {
+      cy.get('q').should('be.visible');
+    });
+  });
+
+  it('Should show quote author', () => {
+    cy.get('blockquote').within(() => {
+      cy.get('cite').should('be.visible');
+    });
+  });
+});
+
+describe('Superblocks and Blocks', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.contains("Get started (it's free)").click({ force: true });
+  });
+
+  it('Has first superblock and block collapsed by default', () => {
+    cy.contains(superBlockNames[0]).should('be.visible');
+    cy.contains(superBlockNames[0]).should(
+      'have.attr',
+      'aria-expanded',
+      'true'
+    );
+
+    cy.contains('Basic HTML and HTML5').should('be.visible');
+    cy.contains('Basic HTML and HTML5').should(
+      'have.attr',
+      'aria-expanded',
+      'true'
+    );
+  });
+
+  it('Has all supeblocks visible but folded (excluding the first one)', () => {
+    cy.contains(superBlockNames[1]).should('be.visible');
+    cy.contains(superBlockNames[1]).should(
+      'have.attr',
+      'aria-expanded',
+      'false'
+    );
+
+    cy.contains(superBlockNames[2]).should('be.visible');
+    cy.contains(superBlockNames[2]).should(
+      'have.attr',
+      'aria-expanded',
+      'false'
+    );
+
+    cy.contains(superBlockNames[3]).should('be.visible');
+    cy.contains(superBlockNames[3]).should(
+      'have.attr',
+      'aria-expanded',
+      'false'
+    );
+
+    cy.contains(superBlockNames[4]).should('be.visible');
+    cy.contains(superBlockNames[4]).should(
+      'have.attr',
+      'aria-expanded',
+      'false'
+    );
+
+    cy.contains(superBlockNames[5]).should('be.visible');
+    cy.contains(superBlockNames[5]).should(
+      'have.attr',
+      'aria-expanded',
+      'false'
+    );
+
+    cy.contains(superBlockNames[6]).should('be.visible');
+    cy.contains(superBlockNames[6]).should(
+      'have.attr',
+      'aria-expanded',
+      'false'
+    );
+
+    cy.contains(superBlockNames[7]).should('be.visible');
+    cy.contains(superBlockNames[7]).should(
+      'have.attr',
+      'aria-expanded',
+      'false'
+    );
+
+    cy.contains(superBlockNames[8]).should('be.visible');
+    cy.contains(superBlockNames[8]).should(
+      'have.attr',
+      'aria-expanded',
+      'false'
+    );
+
+    cy.contains(superBlockNames[9]).should('be.visible');
+    cy.contains(superBlockNames[9]).should(
+      'have.attr',
+      'aria-expanded',
+      'false'
+    );
+
+    cy.contains(superBlockNames[10]).should('be.visible');
+    cy.contains(superBlockNames[10]).should(
+      'have.attr',
+      'aria-expanded',
+      'false'
+    );
```

Comment:
```suggestion
    cy.wrap(superBlockNames.slice(1)).each(name => {
      cy.contains(name)
        .should('be.visible')
        .and('have.attr', 'aria-expanded', 'false');
    });
```
Cypress is clever enough to check all of the assertions in the loop, so we might as well take advantage of that!